src/client/_clientwindow.py

```diff
@@ -870,12 +874,25 @@ def saveWindow(self):
         util.settings.setValue("geometry", self.saveGeometry())
         util.settings.endGroup()
 
+    def show_autojoin_settings_dialog(self):
+        autojoin_channels_list = Settings.get('chat/auto_join_channels')
+        text_of_autojoin_settings_dialog = """
+        Autojoin channels list : """ + str(autojoin_channels_list) + """
+        Enter the list of channels you want to join, separated by ;
+        For example: #poker;#newbie
+        To disable autojoin, leave box empty and press ok
```

Comment:
This gets shown to the user - make sure grammar, interpunction etc. is flawless. IMO we should put dots at sentence end in textboxes.

Reply:
Ok improved it but I'm not an english native speaker so...Maybe I missed something.
I didn't want to add a point after ; though because people may get confused otherwise and add a . after ; even if there is an example... And same for the example.